demo/src/app/app.component.html

```diff
@@ -1,85 +1,119 @@
+        <p class="github-buttons pull-xs-right">
+          <a class="github-button"
+             href="https://github.com/ng-bootstrap/ng-bootstrap"
+             target="_blank"
+             angulartics2On="click"
+             angularticsEvent="Project Stars"
+             angularticsCategory="GitHub"
+             data-icon="octicon-star"
+             data-count-href="/ng-bootstrap/ng-bootstrap/stargazers"
+             data-count-api="/repos/ng-bootstrap/ng-bootstrap#stargazers_count"
+             data-count-aria-label="# stargazers on GitHub"
+             aria-label="Star ng-bootstrap/ng-bootstrap on GitHub">Star</a>
+          <a class="github-button"
+             href="https://github.com/ng-bootstrap/ng-bootstrap/fork"
+             target="_blank"
+             angulartics2On="click"
+             angularticsEvent="Project Forks"
+             angularticsCategory="GitHub"
+             data-icon="octicon-repo-forked"
+             data-count-href="/ng-bootstrap/ng-bootstrap/network"
+             data-count-api="/repos/ng-bootstrap/ng-bootstrap#forks_count"
+             data-count-aria-label="# forks on GitHub" aria-label="Fork ng-bootstrap/ng-bootstrap on GitHub">Fork</a>
+          <a class="github-button"
+             href="https://github.com/ng-bootstrap/ng-bootstrap"
+             target="_blank"
+             angulartics2On="click"
+             angularticsEvent="Project Watch"
+             angularticsCategory="GitHub"
+             data-icon="octicon-eye"
+             data-count-href="/ng-bootstrap/ng-bootstrap/watchers"
+             data-count-api="/repos/ng-bootstrap/ng-bootstrap#subscribers_count"
+             data-count-aria-label="# watchers on GitHub"
+             aria-label="Watch ng-bootstrap/ng-bootstrap on GitHub">Watch</a>
+          <a href="https://github.com/ng-bootstrap/ng-bootstrap"
+            target="_blank"
+            angulartics2On="click"
+            angularticsEvent="Project Source"
+            angularticsCategory="GitHub"
+            aria-label="Homepage">
+           <svg aria-hidden="true" class="octicon octicon-mark-github" height="20" version="1.1" viewBox="0 0 16 16"
+             width="20">
+             <path d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0 0 16 8c0-4.42-3.58-8-8-8z"></path>
+           </svg></a>
+        </p>
+      </div>
     </nav>
   </div>
 </header>
 <div>
   <router-outlet></router-outlet>
 </div>
 <footer class="bd-footer text-muted">
-  <div class="container">
```

Comment:
Footer is big and wierd. I am also not satisfied with this. We should be trimming this.